src/util/defaultRoles.js

```diff
@@ -0,0 +1,120 @@
+export const defaultShopManagerRoles = [
```

Comment:
Please sort the lists in this file alphabetically if possible. If there is a reason the order is significant please add a comment saying so. Having lists that are _almost_ alphabetical with no comment indicating why make it confusing when somebody has to touch this code, wherein they will feel they have to ask around before adding anything to the lists and that they aren't supposed to sort them themselves.

Reply:
There is no reason for it to be un-alphabetized, it can be updated.

I can explain why it is though: it was alphabetized when we originally wrote it, but changed some of the role names, and just changed them directly where they were in the list, and didn't re-sort.